### DISCUSSION TITLE
iPad Release

### DISCUSSION THREAD

**Discussion by talaviram** [plugdata-team/plugdata#1589] [Ideas]
A while ago I was actually able to build this and run it on my iPad. (just made tiny changes to make it compile but it was over a year ago)

While UI was unpolished and stability was mixed bag (since it wasn't the original intention) - it feels like a really fun product I'd love to see on a iPad.

Since the license of the source-code isn't walled-garden (hmmm... AppStore) friendly, it'll be nice if you can consider releasing it to iPad. initially even as a TestFlight seems reasonable.

**Comment by dromer** [comment]
This has already been done, you can find it in the app store: https://apps.apple.com/nl/app/plugdata/id6473122351

**Reply by talaviram** [reply]
Ahh wow! great. too bad I've only looked a year ago!

Thanks!
